README.md

```diff
@@ -1,2 +1,19 @@
 # v5.1-example-box
-Example project to show Truffle v5.1 features
+This box is meant to provide an illustration of several of the new features included with Truffle v5.1.
+
+The first feature to note is Truffle's support of ENS name resolution and address setting. If you own an ENS name, you may use the deployer to set the resolution address using the `deployer.ens.setAddress` method. You can see an example of how this is used in the migration file named `3_set_names.js`. Here, assuming `accounts[0]` owns the names `bob.name` and `alice.name`, we can set the address to which those names resolve when used in place of an address. Note that if you are using a local test net, Truffle will automatically give the necessary permissions for the `from` address to set the resolution address.
```

Comment:
probably worth noting that this "local test net" situation is what the user gets when they run `truffle test` out of the box here